packages/core/src/FrameLoop.ts

```diff
@@ -111,9 +122,11 @@ export class FrameLoop {
   }
 
   /** Advance an animation forward one frame. */
-  advance(config: ActiveAnimation, changes: FrameUpdate[2]): boolean {
-    const time = G.now()
-
+  advance(
+    step: number,
```

Comment:
Let's rename `step` to `dt` to be more clear

Reply:
Done!